neurots/astrocyte/section.py

```diff
@@ -65,7 +65,7 @@ def grow_to_target(start_point, start_direction, target_point, segment_length, p
         )
 
         point += segment_length * direction
-        points.append(point)
+        points.append(point.copy())
```

Comment:
I would prefer if we didn't mutate that point in-place. So instead:
```python
point = point + segment_length * direction
points.append(point)
```
And avoid the in-place mutation altogether.